src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -71,19 +71,19 @@ public Office365ConnectorWebhookNotifier(Run run, TaskListener listener) {
     }
 
     public void sendBuildStartedNotification(boolean isFromPreBuild) {
+        WebhookJobProperty property = (WebhookJobProperty) job.getProperty(WebhookJobProperty.class);
+        if (property == null || property.getWebhooks() == null || property.getWebhooks().size() == 0) {
```

Comment:
isEmpty() ?

Reply:
yes, isEmpty() is a better choice, I can adjust that.